src/astro/utils/database.py

```diff
@@ -10,6 +10,31 @@
 from astro.utils.dependencies import BigQueryHook, PostgresHook, SnowflakeHook
 
 
+def get_database_from_conn_id(conn_id: str) -> Database:
+    """
+    Given a conn_id, identify the database name.
+
+    :param conn_id: Airflow connection ID
+    :type conn_id: str
+    :return: the database this interface relates to (e.g. Database.SQLITE)
+    :rtype: astro.constants.Database enum item
+    """
+    conn_type = BaseHook.get_connection(conn_id).conn_type
+    type_to_db = {
```

Comment:
Unfortunately `Database(conn_type)` doesn't work due to `google_cloud_platform`, which should be mapped to the `Database.BigQuery`. 

I liked the package `aenum`. Do you think it is well maintained and stable enough? Could we log a ticket to refactor to use it?

Reply:
@tatiana I think, we will run into situations where we need to map one db with multiple connection types, it would help to keep our code base clean. However I'm not sure about the stability and maintainability or the package. @kaxil, do you recommend any packages for this situation?